regolith/builders/activitylogbuilder.py

```diff
@@ -32,7 +32,7 @@ class ActivitylogBuilder(LatexBuilderBase):
     """Build CV from database entries"""
 
     btype = "annual-activity"
-    needed_dbs = ['groups', 'people', 'grants', 'proposals', 'institutions',
+    needed_cols = ['groups', 'people', 'grants', 'proposals', 'institutions',
```

Comment:
I think `colls` (short for `collections`) might be better but not sure.  What do you think?

Reply:
can do